### ISSUE TITLE
bug(a11y): announcement text lacks enough contrast

### ISSUE THREAD

**Issue by donroyco** [deckgo/deckdeckgo#1465]
### Affected part of DeckDeckGo

- [x] Site

### Reproduction

Steps to reproduce:

1. Navigate to https://deckdeckgo.com/en/
2. Look/inspect the announcement bar text

### Expected Behavior

To have enough contrast. 

### Actual Behavior

It does not have enough contrast, also stating via Chrome Devtools.

<img width="552" alt="Screenshot 2022-02-04 at 19 27 03" src="https://user-images.githubusercontent.com/1763537/152583676-db6c15ee-a3e8-42d6-b282-74a6028f6b19.png">

### Environment

- Browser(s): Chrome, latests version
- Operating System (e.g. Windows, macOS, Ubuntu):

**Comment by peterpeterparker** [comment]
Thx 👍
